demo/src/main/java/com/example/demo/services/UserServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.example.demo.services;
+
+import com.example.demo.model.User;
+import com.example.demo.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+    @Autowired
+    public UserRepository userRepository;
+
+    @Override
+    public String createUser(User user) {
+        userRepository.save(user);
+        return "User has been created";
+    }
+
+    @Override
+    public String deleteUser(User user) {
```

Comment:
usually delete method returns boolean, check the return value of delete method in repository